ProjectEureka/project-eureka-flutter
# PE-41 | Implement Account Settings Page

What I Did: 

- Created the `Account` settings page which includes pages for changing the **email**, **password**, and **deleting** the account.
- Followed the UI design on Figma with a slight addition of an alert dialog box for `Delete account`.

How to Test: 

- If you haven't already signed up with an account, sign up first then login. 
- Click on the Sidebar Menu and head to `Settings`, then `Account`.

1. For changing the email, type in a new email then hit `Done`. Right after, it should take you back to the `Edit Account` page.

> To test that your email was changed successfully, logout. Then, try signing in with the email you initially signed up with (this should be invalid). Try signing in with the new email you entered (this should successfully log you in). 
**Important:** login to our Firebase console for PE and check that the email you initially signed up with was changed to the one you just created.

2. For changing the password, same steps above apply.
3. For deleting the account, click `Delete` on the alert dialog box. It should take you back to the **Login** page after. 

> Try logging in with the credentials you signed up with (should return email\password not found). 
**Important:** check Firebase to see if your account is deleted.
